sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -201,15 +205,24 @@ const eventHubConsumerClient = new EventHubConsumerClient(consumerGroupName, ehC
 
 const subscription = eventHubConsumerClient.subscribe(
   partitionId, {
-    // In V5 we deliver messages in batches, rather than a single message 
-    // at a time. You can control the batch size via the options passed to the client.
-    processEvents: (messages, context) => {},
+    // In V5 we deliver events in batches, rather than a single message at a time.
+    // You can control the batch size via the options passed to the client.
+    //
+    // If you have asynchronous code running in your callback, it will be awaited before the
```

Comment:
```suggestion
    // If your callback is an async function or returns a promise, it will be awaited before the
```
The current wording makes it sound like just having async code in my callback is enough to have stop receving events until I'm done processing the current batch.